CountryPicker/CountryPicker/CountryPickerController.swift

```diff
@@ -290,18 +290,18 @@ extension CountryPickerController: UITableViewDelegate, UITableViewDataSource {
     
     // MARK: - TableView Delegate
     public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        switch applySearch {
-        case true:
-            let country = filterCountries[indexPath.row]
-            self.callBack?(country)
-            CountryManager.shared.lastCountrySelected = country
-            self.dismiss(animated: false, completion: nil)
-        case false:
-            let country = countries[indexPath.row]
-            self.callBack?(country)
-            CountryManager.shared.lastCountrySelected = country
+        var selectedCountry = countries[indexPath.row]
+        var dismissWithAnimation = true
+        
+        if applySearch {
+            selectedCountry = filterCountries[indexPath.row]
+            dismissWithAnimation = !applySearch
```

Comment:
we can directly assign false to `dismissWithAnimation = false ` here because it's unnecessary to compute it again. It should be always constant to false